site/docs/developer_documentation/devops.md

```diff
@@ -0,0 +1,40 @@
+---
+id: devops
+title: GQCP Devops
```

Comment:
What is the preferred capitalization? Is it "DevOps" or "Devops"?

Reply:
I prefer DevOps. The id is in lowercase.